fr_CA/webapp/src/main/webapp/i18n/all_fr_CA.properties

```diff
@@ -965,3 +965,59 @@ label.dateTimeWithPrecision.day_capitalized = Jour
+
+password_reset_complete_email_html_text = <p>\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n <strong>Password successfully changed.</strong>\n </p>\n\n <p>\n Your new password associated with ${userAccount.emailAddress} has been changed.\n </p>\n\n <p>\n Thank you.\n </p>
+password_reset_complete_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nPassword successfully changed.\n\nYour new password associated with ${userAccount.emailAddress}\nhas been changed
+
+confirm_email_changed_email_html_text = <p>\n Hi, ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n You recently changed the email address associated with\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n Thank you.\n </p>
+confirm_email_changed_email_plain_text = Hi, ${userAccount.firstName} ${userAccount.lastName}\n\nYou recently changed the email address associated with\n${userAccount.firstName} ${userAccount.lastName}\n\nThank you.
+
+first_time_external_email_html_text = <p>\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n <strong>Congratulations!</strong>\n </p>\n\n <p>\n We have created your new VIVO account associated with ${userAccount.emailAddress}.\n </p>\n\n <p>\n Thanks!\n </p>
+first_time_external_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nCongratulations!\n\nWe have created your new VIVO account associated with\n${userAccount.emailAddress}.\n\nThanks!
+
+password_created_email_html_text = <p>\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n <strong>Password successfully created.</strong>\n </p>\n\n <p>\n Your new password associated with ${userAccount.emailAddress} has been created.\n </p>\n\n <p>\n Thank you.\n </p>
+password_created_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nPassword successfully created.\n\nYour new password associated with ${userAccount.emailAddress}\nhas been created.\n\nThank you.
+
+application_error_email_html_text = <p>Une erreur s'est produite sur votre site ${siteName!} à ${datetime!}</p>\n\n <p>\n <strong>URL demandée:</strong> ${requestedUrl!}\n </p>\n\n <p>\n <#if errorMessage?has_content>\n <strong>Message d'erreur:</strong> ${errorMessage!}\n </#if>\n </p>\n\n <p>\n <strong>Trace de la pile</strong> (trace complète disponible dans le log):\n <pre>${stackTrace!}</pre>\n </p>\n\n <#if cause?has_content>\n <p><strong>Causé par:</strong>\n <pre>${cause!}</pre>\n </p>\n </#if>
+
+application_error_email_plain_text = Une erreur s'est produite sur votre site ${siteName!} à ${datetime!}\n\nURL demandée: ${requestedUrl!}\n\n<#if errorMessage?has_content>\n Message d'erreur: ${errorMessage!}\n</#if>\n\nTrace de la pile (trace complète disponible dans le log):\n${stackTrace!}\n\n<#if cause?has_content>\nCausé par:\n${cause!}\n</#if>
+
+application_error_email_subject = Une erreur s'est produite sur votre site ${siteName!}

```

Comment:
```suggestion

```

Reply:
In the French file we have duplicated label keys (firstly with labels translated in French, and then with labels translated in English). The last one are used in the UI (which is wrong).